docker/install.sh

```diff
@@ -1,33 +1,47 @@
 #!/bin/bash
 set -e
 
-# ROS 1 msgs setup
+# Prepare workspace for ROS Noetic dependencies and Autoware messages
+mkdir -p /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai
+
+# Copy Autoware.ai messages into ROS1 workspace
 cp -R /home/carma/autoware.ai/messages /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai/
 cp -R /home/carma/autoware.ai/jsk_common_msgs /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai/
 cp -R /home/carma/autoware.ai/jsk_recognition /home/carma/.base-image/ros1_msgs_ws/src/autoware.ai/
 
-# ROS 2 msgs setup
-mkdir -p ~/.base-image/ros2_msgs_ws/src/autoware.ai
+# ROS1 CARMA message and dependency setup
+cd /home/carma/.base-image/ros1_msgs_ws
+unset ROS_DISTRO
```

Comment:
Why do we need to unset the ROS distro and export the ROS version here? Could we instead source the noetic installation?

Reply:
The ubuntu 22.04 doesn't have noetic installation, as such we don't have the noetic/setup.bash. We unset it temporarily to remove humble so we can build ros1

Reply:
How is ros1 code built without noetic installed?

Reply:
Gotcha, I wasn't familiar with ros-desktop-dev. That's neat!